### PR TITLE
MMI adds the Jwt dropdown component

### PR DESCRIPTION
**Pull request by zone-live**
## Explanation

This PR adds the MMI `JwtDropdown` component under the `ui/components/institutional/` folder
Ref ticket: https://consensyssoftware.atlassian.net/browse/MMI-2651

## Pre-merge author checklist

- [X] I've clearly explained:
  - [X] What problem this PR is solving
  - [X] How this problem was solved

## Pre-merge reviewer checklist

- [X] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [X] PR is linked to the appropriate GitHub issue

### CODE REVIEW

**Review by albertolive — APPROVED** [review]

**Review by albertolive — DISMISSED** [review]

File: ui/components/institutional/jwt-dropdown/jwt-dropdown.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.custody-search-jwt__select {
+  height: 54px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+
+  &-title {
+    @include Paragraph;
+
+    color: var(--scorpion);
```
Comment (darkwing) [inline ui/components/institutional/jwt-dropdown/jwt-dropdown.scss:10]: We no longer use this color in the extension; could it be `var(--color-error-default)`?

File: ui/components/institutional/jwt-dropdown/jwt-dropdown.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import Dropdown from '../../ui/dropdown';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const JwtDropdown = (props) => {
+  const t = useI18nContext();
+  const { currentJwt, jwtList } = props;
+
+  return (
+    <div>
+      <span className="custody-search-jwt__select-title">{t('selectJWT')}</span>
```
Comment (darkwing) [inline ui/components/institutional/jwt-dropdown/jwt-dropdown.js:12]: For this we should use the new `<Text>` design system component with `padding={3}` and `color={Color.___}` values.

File: ui/components/institutional/jwt-dropdown/jwt-dropdown.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import PropTypes from 'prop-types';
+import Dropdown from '../../ui/dropdown';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const JwtDropdown = (props) => {
+  const t = useI18nContext();
+  const { currentJwt, jwtList } = props;
+
+  return (
+    <div>
```
Comment (darkwing) [inline ui/components/institutional/jwt-dropdown/jwt-dropdown.js:11]: Instead of `<div>`, we can use `<Box>`

**Review by darkwing — CHANGES_REQUESTED** [review]
Adding a storybook component will help to ensure design integrity and help with review.

**Review by darkwing — APPROVED** [review]

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**